### DISCUSSION TITLE
Some Ideas.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#110] [Ideas]
1.Hear Everyone Including dead people.
2.Speak in phantom mode (probably no way to do this, but sounds cool).
3.Speak to living players when dead.(also probably wont work and again sounds cool).
3.Jetpack activates automatically when people equip it until it explodes.
4.Normal teleporter(Not Inverse Teleporter) will teleport to any random location in the map when pressing the button. ( like the teleporting to coordinates but automated, For trolling purposes when someone pushes the button XD)

Again all are just suggestions and i realize some of them are not feasible.

**Comment by winstxnhdw** [comment]
> Hear Everyone Including dead people.

All of them seem possible. It's just I am not sure why you'd want to do this.

**Reply by yujispin990** [reply]
E.g im the only one alive and they talking about me and i suddenly respond to them....the reaction would be priceless XD

**Reply by winstxnhdw** [reply]
mmm, but how would you differentiate the dead players and living ones though?

**Reply by yujispin990** [reply]
Of course just to be clear "Hear Everyone" implies that i can hear them clearly anywhere in the map.

**Reply by yujispin990** [reply]
Dead Player List.

**Reply by yujispin990** [reply]
which would be another suggestion xd.

**Reply by winstxnhdw** [reply]
That wouldn't really help? You'd just know who is dead but not who is speaking.

**Reply by yujispin990** [reply]
probably a display that shows which player is dead. but honestly i could just check the ship or scan for dead body or just assume they are dead anyways.

**Reply by winstxnhdw** [reply]
Maybe I can force the dead player HUD to always appear?

**Reply by yujispin990** [reply]
i would need to discern by knowing firstly which voice belongs to who.

**Reply by yujispin990** [reply]
OH WAIT YEAH THAT WORKS! if it's possible that is.

**Reply by winstxnhdw** [reply]
Yeah but I don't know if people would be happy with that

**Reply by yujispin990** [reply]
Ah no worries its just a suggestion, but thank you for showing interest in the idea 😄

**Reply by winstxnhdw** [reply]
I can consider all this, but I'll be focusing on ESP in the coming few days.

**Reply by yujispin990** [reply]
Of course no problem! thank you for even taking it into consideration sir.
